Sources/Paywalls/PaywallCacheWarming.swift

```diff
@@ -97,11 +97,16 @@ final class DefaultPaywallImageFetcher: PaywallImageFetcherType {
 
 private extension Offerings {
 
+    var offeringsToPreWarm: [Offering] {
+        // At the moment we only want to pre-warm the current offering to prevent
+        // apps with many paywalls from downloading a large amount of images
+        return self.current.map { [$0] } ?? []
+    }
```

Comment:
Yeah, this makes sense. Perhaps we could add an API for prewarming a specific offering for anyone who wants to ensure that things are downloaded before getting presented (separate PR though)

Reply:
Yeah @joshdholtz had the same idea. I wonder how discoverable that would be... but if we're asked for it we can definitely add it.